docs/api/ref/Events.md

```diff
@@ -38,9 +66,24 @@ Used when attaching listeners to events.
 
 ## `FDC3EventTypes`
 
+<Tabs groupId="lang">
+<TabItem value="ts" label="TypeScript/JavaScript">
+
 ```ts
 type FDC3EventTypes = "userChannelChanged";
 ```
+</TabItem>
+<TabItem value="dotnet" label=".NET">
+
+```csharp
+public static class Fdc3EventType
+{
+    public const string UserChannelChanged = "UserChannelChanged";
```

Comment:
The PR to fdc3-dotnet was just updated based on feedback.  For developers using the literals rather than the constants themselves, it makes sense to match across implementations for convenience.

```suggestion
    public const string UserChannelChanged = "userChannelChanged";
```